tests/testthat/test_format_dta.R

```diff
@@ -9,7 +9,18 @@ test_that("Export to Stata", {
 test_that("Import from Stata (read_dta)", {
     expect_true(is.data.frame(import("mtcars.dta", haven = TRUE)))
     # arguments ignored
-    expect_warning(is.data.frame(import("mtcars.dta", haven = TRUE, extraneous.argument = TRUE)))
+    expect_warning(import("mtcars.dta", haven = TRUE, extraneous.argument = TRUE))
```

Comment:
I removed `is.data.frame()` from this part of the test since that is not required to generate this warning.